tensorflow/java/build_defs.bzl

```diff
@@ -19,6 +19,7 @@ XLINT_OPTS = [
     "-Xlint:-serial",
     "-Xlint:-try",
     "-Xlint:-classfile", # see b/32750402, go/javac-warnings#classfile
+    "-Xlint:-deprecation", # for exposing deprecated ops
```

Comment:
Can we just not generate Java code for deprecated ops instead of disabling the warning?
Since we don't yet promise API stability, why bother with generating code for deprecated ops?

Reply:
Mmh, that sounds logical... There might be some thinking to do though on how we gonna handle deprecated ops *after* we guarantee that stability.

We will need to keep track of which endpoints were deprecated after the first release and which were deprecated before.

Also, maybe we can enable/disable the generation of deprecated ops using an application flag provided by the bazel rule, so we simply turn that switch on when it is the right time to do so. Then I would leave the `-Xlint:-deprecation` flag there so won't fail when we enable it.

What are you thoughts on this?

Reply:
For now, let's keep it simple and do away with deprecated ones.
Once we commit to stability, we can reconsider (and remove the warnings then, better to keep them on as long as possible than preemptively disabling them)